src/h264_example.c

```diff
@@ -0,0 +1,196 @@
+#include "libuvc/libuvc.h"
```

Comment:
Is there a reason that this can't be combined with the original example.c?

Reply:
combined